e2e/pages/caseList.page.js

```diff
@@ -53,7 +54,7 @@ module.exports = {
   },
 
   locateCase(caseId){
-    return locate(`//ccd-search-result/table//tr[//a[contains(@href,'${caseId}')]]`);
+    return `a[href$='${caseId}']`;
```

Comment:
retryUntilExists does not work with xpath selectors as used I.waitForSelector does not support xpath

Reply:
Forgot to mention this the other day, it is why I like to use wait for element